arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -25,11 +25,11 @@
         "operationId": "UsageAggregates_List",
         "description": "Query aggregated Azure subscription consumption data for a date range.",
         "externalDocs": {
-          "url": "https://msdn.microsoft.com/library/azure/1ea5b323-54bb-423d-916f-190de96c6a3c"
+          "url": "https://docs.microsoft.com/rest/api/commerce/usageaggregates"
         },
         "parameters": [
           {
-            "name": "reportedstartTime",
+            "name": "reportedStartTime",
```

Comment:
Is this broken in the current release? (or does your RP just accept it with any case?)

As this is a parameter, it shouldn't be a breaking change,..